packages/playground/wordpress/src/boot.ts

```diff
@@ -256,3 +257,47 @@ async function installWordPress(php: PHP) {
 			})
 	);
 }
+
+export async function getLoadedWordPressVersion(
+	php: PHP,
+	wordpressRoot: string
+): Promise<string | undefined> {
+	const result = await php.run({
+		code: `<?php
+			if (is_file('${wordpressRoot}/wp-includes/version.php')) {
```

Comment:
`php.documentRoot` should do the trick here. The entire codebase assumes WordPress lives there so, if it doesn't, we have a larger problem.

Reply:
> php.documentRoot should do the trick here. The entire codebase assumes WordPress lives there – if it doesn't, we have a larger problem.

Thanks, I ended up using requestHandler.documentRoot because php.documentRoot is marked as deprecated.